MyPlayground.playground/Pages/main.xcplaygroundpage/Contents.swift

```diff
@@ -3,35 +3,45 @@
  
  # Functions
  
- ### Readings associated with this lab
+ ## Readings associated with this lab
  
  * [Functions](https://github.com/learn-co-curriculum/swift-functions-readme)
 
  */
 /*: question1
- ### 1. Question making the student think of how best to structure a function (or this first one can be VERY simple in just having them get used to the syntax of typing a function)
+ ### 1. Write a function called `helloWorld()` that prints "Hello, world!" to the console. Then call it to see your string printed to the playground console.
  */
 // write your code here
 
 
 
 
 /*: question2
- ### 2. Another question about functions.. weeee
+ ### 2. Write a function that declares a constant inside the function's body and prints it to the console. Then call it to see your string printed to the playground console.
  */
```

Comment:
This wording can be changed a little. I can see some students looking to deep into it and get lost in what it is they actually have to do.

Being slightly more explicit:

> Write your own function, where inside your function's body you declare a constant and then print that constant to the console. Call this function to see your string printed to the playground console.